### PR TITLE
Fix powershell script for nuget update

### PR DESCRIPTION
**Pull request by SaurabhMSFT**
## Description

The powershell script that update nuget config file should be run in the correct directory.

### CODE REVIEW

File: build/template-Build-run-tests-sign.yml
Comment (iNinja) [inline build/template-Build-run-tests-sign.yml:1]: Lines 51, 52, and 54 appear to be debug information.

Could we clean that and just keep the `workingDirectory` change?

Comment (SaurabhMSFT) [reply build/template-Build-run-tests-sign.yml:1]: That is temporary changes to debug the build.

File: NuGet.Config [hunk 1]
```diff
@@ -2,6 +2,6 @@
 <configuration>
   <packageSources>
     <clear />
-	<add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+  <add key="IDDP" value="https://identitydivision.pkgs.visualstudio.com/_packaging/IDDP/nuget/v3/index.json" />
```
Comment (brentschmaltz) [inline NuGet.Config]: Our external customers will not be able to reach these packages and their local builds will fail.

File: build/template-Build-run-tests-sign.yml [hunk 2]
```diff
@@ -47,12 +47,14 @@ steps:
   displayName: 'Strong Name Bypass'
 
 - powershell: |
+    $q = (dotnet nuget list source --format Short)
```
Comment (iNinja) [inline build/template-Build-run-tests-sign.yml]: This line is introduced, but on line 51 we still call the command again to verify whether it contains NuGet's index.

Comment (SaurabhMSFT) [reply build/template-Build-run-tests-sign.yml]: this is to print the first run message. let me see if i can avoid it using an env variable.

**Comment by jennyf19** [comment]
did you run a release build off of this branch to ensure that still works?

**Comment by SaurabhMSFT** [comment]
> did you run a release build off of this branch to ensure that still works?

Yes, it works https://dev.azure.com/IdentityDivision/IDDP/_build/results?buildId=1395135&view=results

**Pull request closed without merging**